modules/serverless-kafka-source-odc.adoc

```diff
@@ -3,7 +3,7 @@
 // * serverless/event_sources/serverless-kafka-source.adoc
 
 [id="serverless-kafka-source-odc_{context}"]
-= Creating a Kafka event source by using the web console
+= Creating a Kafka event source using the web console
```

Comment:
I think that "by using" is actually the preferred wording, according to the IBM style guide.

> Do not omit articles and prepositions that can increase the clarity of a sentence.
 Incorrect: Unload the file using the ULOAD utility. 
 Correct: Unload the file by using the ULOAD utility